### PR TITLE
Amen chw/issue2

### PR DESCRIPTION
**Pull request by AmenCHW**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2045,10 +2045,10 @@
     redux-thunk "^2.4.1"
     reselect "^4.1.5"
 
-"@remix-run/router@1.1.0":
-  version "1.1.0"
-  resolved "https://registry.yarnpkg.com/@remix-run/router/-/router-1.1.0.tgz#b48db8148c8a888e50580a8152b6f68161c49406"
```
Comment (hudahamid) [inline yarn.lock:2050]: does it require to do this edition?

**Review by hudahamid — APPROVED** [review]
looks good to me, great
white
but there is white space after the footer on the medium screen, if you check it

**Comment by Danah-94** [comment]
LGTM , Don't forget latter to add a link to the icons in the footer

**Review by Danah-94 — COMMENTED** [review]
LGTM, don't forget latter to add a link to the icons in the footer

**Review by Danah-94 — APPROVED** [review]
LGTM, latter  don't forget to add a link to each icon in the footer

**Pull request merged**